.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: WWvVcbssogMdeAbsS7NM32Zm1bEizAgcX
```

Comment:
You might not want to put this here. There should be a way to add it to the environment variable of your CI platform.